wom/src/main/scala/wom/views/GraphPrint.scala

```diff
@@ -0,0 +1,200 @@
+package wom.views
+
+import java.util.concurrent.atomic.AtomicInteger
+
+import cats.implicits._
+import cats.Monoid
+import wom.callable.ExecutableCallable
+import wom.graph.GraphNodePort.{ConditionalOutputPort, OutputPort, ScatterGathererPort}
+import wom.graph.expression.ExpressionNode
+import wom.graph._
+import wom.types.WomType
+import wom.views.GraphPrint._
+
+final class GraphPrint(executableCallable: ExecutableCallable) {
+
+  def dotString = WorkflowDigraph(executableCallable.name, listAllGraphNodes(executableCallable.graph, new AtomicInteger(0), Map.empty)).dotString
+
+  implicit val nodeAndLinkMonoid: Monoid[NodesAndLinks] = cats.derived.MkMonoid[NodesAndLinks]
```

Comment:
possibly a note for what this is doing? I'm okay looking at magic, but a hint would go a long way :)

Reply:
I added a comment. This line is just telling the `cats` library that `NodesAndLinks` instances can be added together. It doesn't do anything active, but allows the use of `foldMap` later on in the file.